crawl-ref/source/beam.cc

```diff
@@ -4624,8 +4628,11 @@ void bolt::knockback_actor(actor *act, int dam)
     coord_def newpos = oldpos;
     for (int dist_travelled = 0; dist_travelled < distance; ++dist_travelled)
     {
-        if (x_chance_in_y(weight, roll))
-            continue;
+        // add 1 guaranteed distance to blowback for differential expansion
+        // mprf("weight: %i  roll: %i  distance: %i dist_travelled: %i",weight,roll,distance,dist_travelled);
+        if (origin_spell != SPELL_DIFFERENTIAL_EXPANSION || dist_travelled != 0)
+            if (x_chance_in_y(weight, roll)) 
+                    continue;
```

Comment:
Not necessarily needed because the next line is one if statement that does one line, but if (origin_spell != SPELL_DIFFERENTIAL_EXPANSION || dist_travelled != 0) should have curly braces to make it easier to tell where it ends since more than one line are under it.